### PR TITLE
Remove step 4

### PR DESCRIPTION
**Pull request by gburnhill**

### CODE REVIEW

File: UserService/UserService.Repo/Repository.cs [hunk 1]
```diff
@@ -681,7 +651,7 @@ public Task<List<UserRegistrationStep>> GetIncompleteRegistrationStatusAsync(Can
                    UserId = g.Key,
                    DateCompleted = g.Max(x=> x.DateCompleted)
                })
-               .Where(a=> a.RegistrationStep!=5).ToList());
+               .Where(a=> a.RegistrationStep<4).ToList());
```
Comment (mike-f50) [inline UserService/UserService.Repo/Repository.cs]: I think this method is to check for anyone who hasn't yet completed registration and done Yoti - so unless we're making Yoti step 4 - which I don't think we should - I think this ought to stay as is.

**Comment by gburnhill** [comment]
Hmm... I thought it was to decide if you get to go to your profile page or whether you’re booted back to Reg flow...

Sent from my iPhone

Geoff Burnhill | Doctor | Musician

On 12 Oct 2020, at 13:01, mike-f50 <notifications@github.com> wrote:

﻿

@mike-f50 commented on this pull request.

________________________________

In UserService/UserService.Repo/Repository.cs<https://github.com/HelpMyStreet/user-service/pull/78#discussion_r503246986>:

> @@ -681,7 +651,7 @@ public Task<List<UserRegistrationStep>> GetIncompleteRegistrationStatusAsync(Can
                    UserId = g.Key,
                    DateCompleted = g.Max(x=> x.DateCompleted)
                })
-               .Where(a=> a.RegistrationStep!=5).ToList());
+               .Where(a=> a.RegistrationStep<4).ToList());


I think this method is to check for anyone who hasn't yet completed registration and done Yoti - so unless we're making Yoti step 4 - which I don't think we should - I think this ought to stay as is.

**Comment by gburnhill** [comment]
Oh I see.

Yes!

Sent from my iPhone

Geoff Burnhill | Doctor | Musician

On 12 Oct 2020, at 13:24, Geoff Burnhill <Geoff@factor-50.co.uk> wrote:

﻿ Hmm... I thought it was to decide if you get to go to your profile page or whether you’re booted back to Reg flow...

Sent from my iPhone

Geoff Burnhill | Doctor | Musician

On 12 Oct 2020, at 13:01, mike-f50 <notifications@github.com> wrote:

﻿

@mike-f50 commented on this pull request.

________________________________

In UserService/UserService.Repo/Repository.cs<https://github.com/HelpMyStreet/user-service/pull/78#discussion_r503246986>:

> @@ -681,7 +651,7 @@ public Task<List<UserRegistrationStep>> GetIncompleteRegistrationStatusAsync(Can
                    UserId = g.Key,
                    DateCompleted = g.Max(x=> x.DateCompleted)
                })
-               .Where(a=> a.RegistrationStep!=5).ToList());
+               .Where(a=> a.RegistrationStep<4).ToList());


I think this method is to check for anyone who hasn't yet completed registration and done Yoti - so unless we're making Yoti step 4 - which I don't think we should - I think this ought to stay as is.

File: UserService/UserService.Repo/Repository.cs (hunk 1 above)
Comment (jawwadmukhtargmail) [reply UserService/UserService.Repo/Repository.cs]: my gut feeling is that it should stay as it is.

Comment (jawwadmukhtargmail) [reply UserService/UserService.Repo/Repository.cs]: it should stay as .Where(a=> a.RegistrationStep!=5).ToList());

**Review by mike-f50 — APPROVED** [review]

**Pull request merged**